### ISSUE TITLE
Max ACC~= 0.7 in Cifar100 and  Max ACC~= 0.39 in Cifar10

### ISSUE THREAD

**Issue by tvmsouza** [vector-1127/DAC#14]
Hello vector,

i tested the code in cifar100 w/ 20 clusters and the max acc that i achieve was ~0.7 and in the cifar10 w/ 10 clusters  was ~0.39. There is some part of the experiment that is not described in the code and in the paper.

I just ran the code like is in the github, and dont achieved the results showed in the paper.

How you set the first wheights?  There is something related to this that you forgot?

I subtracted the means of the images in cifar10 and 100 and the results improved to this values, that are much better than before. 

There is something more in data that i need to do?

Thanks

**Comment by tvmsouza** [comment]
What is your learning rate decay after the initial stage of training?

**Comment by divamgupta** [comment]
Any comments @vector-1127 ?

**Comment by divamgupta** [comment]
@tvmsouza I am not able to surpass 0.22 accuracy in cifar10. Any suggestions?
Thanks

**Comment by vector-1127** [comment]
Hello，
@tvmsouza @divamgupta ，

Thanks for attention and sorry for the delayed response.

I have reported the code of CIFAR10 to avoid the difficulty of reproduction.

Welcome to discuss and exchange.

**Comment by vector-1127** [comment]
Hi, 
@tvmsouza ,

Preprocessing is beneficial to improve the performance of the algorithm. Just like the 31 to 33 lines in https://github.com/vector-1127/DAC/blob/master/Datasets_description/read_data.py, i.e.,
#####################################
data[:,0,:,:] -= np.mean(data[:,0,:,:])
data[:,1,:,:] -= np.mean(data[:,1,:,:])
data[:,2,:,:] -= np.mean(data[:,2,:,:])
#####################################

**Comment by dipanjan06** [comment]
Hi,
I ran the cifar10_2.py code and get acc 21% after 50 epoch . For the data preprocessing I have used code mentioned in https://github.com/vector-1127/DAC/blob/master/Datasets_description/read_data.py. Am I missing anything ? can you please help to reproduce the result mentioned in the paper.
